games/static/games/proto.js

```diff
@@ -1,24 +1,48 @@
 class Game {
-    constructor() {
-
+    constructor(setup) {
+        this.setup = setup
+    }
+    start() {
+        let alienMovementId
+        let grid = document.querySelector('.grid')
+        let board = new Board(grid, 15, this.setup)
+        board.draw(this.setup)
+        this.controls(board)
+        alienMovementId = setInterval(() => { 
+            if (board.alienLocations.length === 0) clearInterval(alienMovementId)
+            board.moveAliens() }, 500)
+        const startButton = document.querySelector('#level1')
```

Comment:
shouldnt this go inside `this.controls` ?
i feel like there are different level of abstractions

Reply:
this is still nothing like it should be i did it just so i could test the functions we were working on in Board class